lib/pages/edit_profile.dart

```diff
@@ -109,6 +109,7 @@ class EditProfileState extends State<EditProfile> {
     if (_formState.currentState.validate() && entity.photoUrl != null) {
       await saveForm();
     }
+    Navigator.pop(context);
```

Comment:
We use Navigate.of(context).pop().
Besides it's on the wrong line. Pls check it again

Reply:
beim onPressed wir submitForm aufgerufen. Deswegen habe ich es dort reingetan. Was genau ist daran falsch?
Zwecks Navigate.of(context).pop() und Navigator.pop(context): ich habe beides ausprobiert und keinen Unterschied gemerkt

Reply:
Es gibt keinen unterscheid. Der code soll nur an allen stellen gleich sein. Und wir verwenden im Source code nirgendswo Navigator.pop(context)